apps/ima/index.js

```diff
@@ -33,11 +36,74 @@ module.exports = {
       behaviours: SaveFormSession,
       fields: ['who-are-you'],
       locals: { showSaveAndExit: true },
-      next: '/confirm', // TO BE UPDATED AS STEPS ARE ADDED
+      next: '/human-rights-claim', // TO BE UPDATED AS STEPS ARE ADDED
       backLink: 'current-progress'
     },
+    '/human-rights-claim': {
+      behaviours: SaveFormSession,
+      fields: ['human-rights-claim'],
+      locals: { showSaveAndExit: true },
+      forks: [
+        {
+          target: '/human-rights-family',
+          condition: {
+            field: 'human-rights-claim',
+            value: 'yes'
+          },
+          continueOnEdit: true
```

Comment:
This is not needed as it is already on line 56. the `continueOnEdit` property relates to the step not the fork.

Reply:
Removed this line.